sys/vane/ford.hoon

```diff
@@ -1792,6 +1793,7 @@
   ::    +build-receipt algorithms to the ford state.
   ::
   ++  execute
+    !:
```

Comment:
Is this one strictly necessary? Looks like it'd already be set by the top-level `!:`.

Reply:
Well, this only gets called by `+execute-loop`, which is `!.`ed. Not sure which takes precedence, so I figured I'd err on the safe side and explicitly enable debugging here.

Reply:
Per a quick test in dojo, it should be safe to remove:

```
> =<  foo  |%  ++  foo  !.  bar  ++  bar  !!  --
ford: %ride failed to execute:
```

```
> !:  =<  foo  |%  ++  foo  !.  bar  ++  bar  !!  --
/~zod/home/~2019.2.5..22.34.32..70c1:<[1 5].[1 51]>
/~zod/home/~2019.2.5..22.34.32..70c1:<[1 9].[1 12]>
/~zod/home/~2019.2.5..22.34.32..70c1:<[1 27].[1 34]>
/~zod/home/~2019.2.5..22.34.32..70c1:<[1 45].[1 47]>
ford: %ride failed to execute:
```

```
> !:  =<  foo  |%  ++  foo  !.  bar  ++  bar  !:  !!  --
/~zod/home/~2019.2.5..22.34.52..c489:<[1 5].[1 55]>
/~zod/home/~2019.2.5..22.34.52..c489:<[1 9].[1 12]>
/~zod/home/~2019.2.5..22.34.52..c489:<[1 27].[1 34]>
/~zod/home/~2019.2.5..22.34.52..c489:<[1 45].[1 51]>
/~zod/home/~2019.2.5..22.34.52..c489:<[1 49].[1 51]>
ford: %ride failed to execute:
```